docs/getting-started.md

```diff
@@ -70,18 +70,12 @@ For information around how to set up React Native, see the [React Native Getting
   <div id="stable">
 The latest and recommended release to use.
 
-## Install React native

```

Comment:
We should have a step to install react native, just point to React Native getting stated guide from that section.

Reply:
From my understanding of npx, it also installs React Native for you, just locally instead of globally. @acoates-ms is that correct?

Reply:
Yeah, npx will install react-native to a local cache similar to install -g, but unlike install -g, it wont pollute your global state.

Reply:
Very cool. The update looks good in that case.